community/doc_guidelines/clis.md

```diff
@@ -0,0 +1,77 @@
+# CLI Help Guidelines
+
+Use these guidelines for Splinter command usage statements (the output of `-h`
+and `--help`).
+
+## General
+
+* Keep the usage statement as short as possible. Put more extensive information 
+  on the man page.
+
+* Use the term "CLI" (command line interface) only for the general interface as
+  a whole. Each item is a command or subcommand. For example:
+
+  - The `splinterd` CLI lets you configure and run the Splinter daemon from the
+   command line.
+
+  - Use the `splinter keygen` command to generate user keys.
+
+  - Use the `splinter keygen` command to generate user keys.
+    <br><br>
+
+* If you choose to say "command line interface", always use the full term
+  (don't omit "interface").
+
+* Avoid starting a sentence with a word that is normally lowercase. For example, 
+  Change "splinterd" to "Splinter daemon".
+
+* Follow the [Splinter capitalization guidelines](capitalization.md).
+
+## Description Section
+
+* Briefly describe the command (what it does and why someone would use it).
+
+* Start the description with a capital letter (noun, verb, or adjective).
+  Do not start with "A", "An", or "The".
+
+* If the summary starts with a verb, use the 3rd-person singular (add an
+  "s"). For example: "Displays", "Manages", "Checks".
+
+* Do not add a period at the end of the description.
+
+## FLAGS and OPTIONS Sections
+
+* Note this terminology:
+  - A _flag_ occurs alone (without an argument). Do not use the term "switch".
+  - An _option_ has at least one _argument_.
+  - For equivalent items in the configuration file, use the term "configuration
+    setting" or "setting".
+
```

Comment:
adding `<br><br>` here would provide better formatting of this list

Reply:
Done.